class AccountNotFoundException(Exception):
    pass


class OutOfBoundsException(Exception):
    pass
